scripts/generateNFCsForAllRegistrations.js

```diff
@@ -0,0 +1,99 @@
+import * as dotenv from "dotenv";
+import {
+  DynamoDBClient
+} from "@aws-sdk/client-dynamodb";
+import {
+  PutCommand,
+  ScanCommand
+} from "@aws-sdk/lib-dynamodb";
+import {humanId} from 'human-id'
+
+dotenv.config({
+  path: "../.env"
+});
+
+
+const awsConfig = {
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  region: "us-west-2",
+};
+
+const client = new DynamoDBClient(awsConfig);
+
+const create = async function (item, table) {
+  try {
+    const params = {
+      Item: item,
+      TableName: table + (process.env.ENVIRONMENT || ""),
+      ConditionExpression: "attribute_not_exists(id)"
+    };
+    const command = new PutCommand(params);
+    const res = await client.send(command);
+    return res;
+  } catch (err) {
+    console.error(err);
+    throw err;
+  }
+};
+
+const generateNFCsForAllRegistrations = async (eventID, year) => {
```

Comment:
Can we update the function name to be something like `generateNFCsForEventRegistrations` to make it clear that it's only affecting a single event instead of all events